.github/workflows/check-pr-title.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       # check PR title
       - uses: deepakputhraya/action-pr-title@master
         with:
-          regex: '([a-z])+(\(([a-z\-_ ])+\))?!?: [a-z]([a-zA-Z-\.\d \(\)\[\]#_,])+$' # Regex the title should match.
+          regex: '([a-z])+(\(([0-9a-z\-_, ])+\))?!?: [a-z]([a-zA-Z-\.\d \(\)\[\]#_,])+$' # Regex the title should match.
```

Comment:
should comma and space be allowed in the scope of a PR title? and do we still need the PR title workflow? (at times I'd wish we would have it in all repos... but having it only in this one seems a bit weird to me)

Reply:
In the present case, it's necessary. (Otherwise I wouldn't have done it :-) 
I find it good to have the PR title workflow, so we get clean and consistent commit messages in the main branch. Surely, other repos would profit from such a workflow!